src/borg/testsuite/__init__.py

```diff
@@ -51,6 +52,21 @@
 if sys.platform.startswith("netbsd"):
     st_mtime_ns_round = -4  # only >1 microsecond resolution here?
 
+rejected_dotdot_paths = (
+    "..",
+    "../",
+    "/..",
+    "/../",
+    "/../etc",
+    "/../etc/",
+    "etc/..",
+    "/etc/..",
+    "//etc/..",
+    "etc//..",
+    "etc/..//",
+    "foo/../bar",
```

Comment:
maybe one of the above cases already triggers the same check, but guess we should have something like `../etc/passwd` (or shadow) here, just to catch precisely an attack vector.

Reply:
Certainly won't hurt, added.